### ISSUE TITLE
Where is the callback for early stopping when no improvements are seen?

### ISSUE THREAD

**Issue by Happily-Coding** [lucacarniato/Optuna-cross-validation-search#1]
Hi, this does look interesting, but on the readme.md at one point it says: "During cross-validation of a keras model, a callback function is used to stop fitting the model when the validation accuracy does not improve after 50 epochs."
But i don't see any use of  callbacks. Where are they?
